### PR TITLE
Introduce exception : PathError, FormatError

### PR DESCRIPTION
**Pull request by mithunbharadwaj**

### CODE REVIEW

**Comment by mithunbharadwaj** [comment]
@ateska 

output from console:

```
{
    "result": "NOT-FOUND",
    "message": "Invalid path '/Templates/MISSING.html'.",
    "uuid": "8bf399be-e8cd-4b47-b72a-62668af23792"
}
```

File: asabiris/exceptions.py [hunk 1]
```diff
@@ -1,2 +1,37 @@
...
+
+	Equivalent to HTTP 400 Bad-request.
+	"""
+
+	def __init__(self, message=None, *args, format=None):
+		self.Format = format
+		if message is not None:
+			super().__init__(message, *args)
+		elif format is not None:
+			message = "Unsupported template format {!r}.".format(format)
```
Comment (mithunbharadwaj) [inline asabiris/exceptions.py:33]: @ateska Should this message be an "Unsupported conversion type"?

File: asabiris/exceptions.py [hunk 2]
```diff
@@ -1,2 +1,37 @@
...
+		if message is not None:
+			super().__init__(message, *args)
+		elif path is not None:
+			message = "Invalid path {!r}.".format(path)
+			super().__init__(message, *args)
+		else:
+			super().__init__(message, *args)
+
+
+class FormatError(Exception):
```
Comment (mithunbharadwaj) [inline asabiris/exceptions.py:22]: @ateska I have a new exception when there is a presence of format html/pdf while dealing with templates.

```
{
    "to": ["foo@example.com"],
     "body":{
      "template":"/Templates/Emaifl/hello.md",
      "params":{
         "name":"Iris"
      }
   },
    "attachments":[
        {
            "template":"/Templates/Emfail/hello.md",
            "params":{
                "name":"Iris"
            },       
            "format":"pdf"
        }
      ]
   }

```

File: asabiris/exceptions.py (hunk 1 above)
Comment (ateska) [reply asabiris/exceptions.py:33]: "Unsupported format"

File: asabiris/exceptions.py (hunk 2 above)
Comment (ateska) [reply asabiris/exceptions.py:22]: ok

**Review by ateska — APPROVED** [review]

**Review by ateska — APPROVED** [review]

**Review by ateska — APPROVED** [review]

**Pull request merged**